README.md

```diff
@@ -1,32 +1,186 @@
+
+-----
+
+## Getting in touch with us
+While we are primarily reachable on our <a href="https://discourse.pi-hole.net/">Discourse User Forum</a>, we can also be found on a variety of social media outlets. **Please be sure to check the FAQ's** before starting a new discussion, as we do not have the spare time to reply to every request for assistance.
+
+<ul>
+  <li><b><a href="https://discourse.pi-hole.net/c/faqs">Frequently Asked Questions</a></b></li>
+  <li><b><a href="https://github.com/pi-hole/pi-hole/wiki">Pi-hole Wiki</a></b></li>
+  <li><b><a href="https://discourse.pi-hole.net/c/feature-requests?order=votes">Feature Requests</a></b></li>
+  <li><a href="https://discourse.pi-hole.net/">Discourse User Forum</a></li>
+  <li><a href="https://www.reddit.com/r/pihole/">Reddit</a></li>
+  <li><a href="https://twitter.com/The_Pi_Hole">Twitter</a></li>
+  <li><a href="https://www.facebook.com/ThePiHole/">Facebook</a></li>
+  <li><a href="https://gitter.im/pi-hole/pi-hole">Gitter</a> (Real-time chat)</li>
+  <li><a href="https://www.youtube.com/channel/UCT5kq9w0wSjogzJb81C9U0w">YouTube</a></li>
+</ul>
+
+# Features
+
+## Mobile friendly interface
+<p align="center">
+<img src="https://pi-hole.github.io/graphics/Screenshots/mobile-friendly.png" height="300" alt="Mobile friendly"></a>
+</p>
+
+## Password protection
+<p align="center">
+<img src="https://pi-hole.github.io/graphics/Screenshots/password-protection.png" height="400" alt="Password protection"></a>
+</p>
+
+## Detailed graphs and doughnut charts
+<p align="center">
+<img src="https://pi-hole.github.io/graphics/Screenshots/piecharts.png" alt="Pie charts"></a>
+</p>
+
+## Top lists of domains and clients
+<p align="center">
+<img src="https://pi-hole.github.io/graphics/Screenshots/topdomains-clients.png" alt="Top domains/top clients"></a>
+</p>
+
+## A filterable and sortable query log
+<p align="center">
+<img src="https://pi-hole.github.io/graphics/Screenshots/query-log-sorted.png" alt="Query log"></a>
+</p>
+
+## An audit log
+<p align="center">
+<img src="https://pi-hole.github.io/graphics/Screenshots/audit-log.png" alt="Pi-hole Web interface"></a>
+</p>
+
+## Long Term Statistics to view data over user defined time ranges
+<p align="center">
+<img src="https://pi-hole.github.io/graphics/Screenshots/long-term-stats.png" alt="Long-term stats"></a>
+</p>
+
+## A built-in debugger
+<p align="center">
+<img src="https://pi-hole.github.io/graphics/Screenshots/debug.png" alt="Debugger"></a>
+</p>
+
+## Black and white lists
+<p align="center">
+<img src="https://pi-hole.github.io/graphics/Screenshots/blacklist.png" alt="Blacklist"></a>
+</br></br>
+<img src="https://pi-hole.github.io/graphics/Screenshots/whitelist.png" alt="Whitelist"></a>
+</p>
+
+## The ability to easily manage and configure Pi-hole features
+<p align="center">
+<img src="https://pi-hole.github.io/graphics/Screenshots/dashboard-pihole-repo.png" alt="Settings"></a>
```

Comment:
This image isn't showing up for me

Reply:
Oops.  I never finished that one.  It's a gif of the different settings page.  Good catch.  I'll fix that